### PR TITLE
Modify AboutUs

### PR DESCRIPTION
**Pull request by YuZizhen**
Modyfy the AboutUs.md to include the information about Zizhen

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -7,17 +7,14 @@ We are a team based in the [School of Computing, National University of Singapor
...
+<img src="images/yuzizhen.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/YuZizhen)]
 
-* Role: Project Advisor
-
-### Koh Guan Zeh
```
Comment (Javiery3889) [inline docs/AboutUs.md:20]: I think you accidentally deleted this line

**Review by Javiery3889 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (YuZizhen) [reply docs/AboutUs.md:20]: AHHH ok i have added it back SORRY

**Review by Javiery3889 — APPROVED** [review]
LGTM

**Comment by Arixeyeion** [comment]
LGTM

**Review by Arixeyeion — APPROVED** [review]
LGTM!

**Pull request merged**